app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -28,102 +31,58 @@ keywords:
 
-Most teacher training courses are non-salaried. This means that you have to pay course fees and will not earn a salary while you train.
-
-Fees are usually around £9,250, and can be higher for [international students](/non-uk-teachers/train-to-teach-in-england-as-an-international-student).
-
-If you train to teach certain secondary subjects, you may be eligible for a [scholarship or bursary](/funding-and-support/scholarships-and-bursaries).
-
-You can apply for a student loan regardless of the course you choose, even if you've had one previously for other postgraduate training.
-
-[Learn about how you could fund your teacher training](/funding-and-support).
-
-### Teacher training courses
-
-Your teacher training course might be provided by a university, school, college or group of schools.
-
-As part of selecting your teacher training course, you will need to decide if you want to train to teach at a primary or secondary level. [Learn about deciding who to teach](/train-to-be-a-teacher/who-do-you-want-to-teach).
-
-Some courses have different course structures and have placements in different kinds of schools, such as special schools. Talk to your teacher training provider to find out what they can offer you.
-
-[Find teacher training courses](https://www.find-postgraduate-teacher-training.service.gov.uk/).
-
-### Choose your provider
-
-One of the best ways to decide between providers is to contact them.
-
-Ask them what a typical day looks like, how you’ll be supported, and if they have any open days or [teacher training events](/events) you can attend.
-
-You might also want to ask:
-
-* how they arrange theoretical learning and school placements
-* who your mentor will be, and how often you'll meet them
-* how many schools you'll be doing school placements in and how long each will last
-* the starting and finishing dates of the course
-
-### Find out where you’ll spend your time
-
-You should talk to teacher training providers about where your school placements and theoretical learning will be. The address of your teacher training provider may not be where you spend most of your time. For example, your teacher training provider might be a university, but you’ll spend most of your time in placement schools.
-
-[Find out more about what your teacher training will be like](/train-to-be-a-teacher/initial-teacher-training).
-
-## Salaried teacher training
-
-Salaried courses are sometimes available for specific candidates, such as if you’re working as an unqualified teacher or teaching assistant. These include School Direct salaried courses, postgraduate teaching apprenticeships and teacher training delivered by Teach First.
-
-These are in high demand and very competitive. Places fill up quickly.
-
-You should check your eligibility with the teacher training provider before you apply.
-
-[Find out more about salaried teacher training](/funding-and-support/salaried-teacher-training).
+* you train to teach certain subjects
```

Comment:
Decided to do one big link to the funding section rather than lots of little links to the specific content pages - mainly did this to reduce the link load, but actually not sure which approach is best?